plans/splitbrain/manifest.toml

```diff
@@ -0,0 +1,33 @@
+name = "network"
+
+[defaults]
+builder = "exec:go"
+runner = "local:exec"
+
+[builders."docker:go"]
+enabled = true
+
+[builders."exec:go"]
+enabled = true
+
+[runners."local:docker"]
+enabled = true
+
+[runners."local:exec"]
+enabled = true
```

Comment:
Should probably disable the `local:exec` runner since this test requires the sidecar.